pkg/costmodel/cluster.go

```diff
@@ -346,6 +346,10 @@ func ClusterNodes(cp cloud.Provider, client prometheus.Client, duration, offset
 		nodeMap[key].CPUCost += cpuCost
 		nodeMap[key].NodeType = nodeType
 	}
+	partialCPUMap := make(map[string]float64)
+	partialCPUMap["e2-micro"] = 0.25
+	partialCPUMap["e2-small"] = 0.5
+	partialCPUMap["e2-medium"] = 1.0
```

Comment:
@AjayTripathy looks like you decided against the f1- and g1- entries, so we're handling that above?

Reply:
I believe we are on the price emission side. I don't want to blow up scope. e2 is very high priority given that new clusters are provisioned with e2. I have added a sprint task to test this fully with f1/g1